docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
       {
         name: 'disabled',
         type: 'boolean',
+        description: `Used to disable the entire SelectList. See the [states](#States) variant to learn more.`,
         defaultValue: 'false',
       },
       {
         name: 'errorMessage',
         type: 'string',
-        href: 'exampleWithError',
+        description: `Used to communicate error information to the user. Be sure to localize the text. See the [states](#States) variant to learn more.`,
       },
       {
         name: 'helperText',
         type: 'string',
-        description: 'More information about how to complete the form field',
-        href: 'helperText',
+        description: `Used to provide more information about the form field. Be sure to localize the text. See the [helper text](#Helper-text) variant to learn more.`,
       },
       {
         name: 'id',
         type: 'string',
+        description:
+          'A unique identifier to connect the underlying `<select>` with the associated label',
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'label',
         type: 'string',
+        description: 'The label shown above the input. Be sure to localize the label.',
       },
       {
         name: 'name',
         type: 'string',
-        href: 'basicExample',
+        description: 'Used to specify the name of the control.',
       },
       {
         name: 'onChange',
-        type: '({ event: SyntheticInputEvent<>, value: string }) => void',
+        type: '({| event: SyntheticInputEvent<>, value: string |}) => void',
+        description: 'Callback triggered when the user makes any changes',
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'options',
-        type: 'Array<{ label: string, value: string, disabled?: boolean }>',
+        type: 'Array<{| label: string, value: string, disabled?: boolean |}>',
+        description:
+          'The options displayed in the dropdown list. Note that ``disabled`` here is used to disable a single option. Be sure to localize the label.',
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'placeholder',
         type: 'string',
-        href: 'basicExample',
+        description:
```

Comment:
I think the placeholder is broken. If passed, options can't be selected. Maybe it requires a specific implementation like value?

Reply:
If no variant is added for this, it's hard to catch issues.

Reply:
I don't believe it's broken, but placeholder and value must be used together as a controlled component.

Reply:
Any suggestions on how to document this? Is this something we could backlog?

Reply:
We should probably add a variant that demonstrates how SelectList works as a controlled component. It'll need a parent component that maintains state, something roughly like
```
function Parent() {
const [value, setValue] = React.useState('');

return (
  <SelectList
    placeholder="foo"
    onChange={setValue}
    value={value}
  />
)
}
```

Reply:
OK, few things:
* I added a variant for onChange, but the card title enforces sentence case, so the title looks like Onchange. :( 
* I am not at all equipped to write a description for this example. :D
* Does `onChange` belong in the "Variant" category? Seems more functional, but I'll defer to y'all.

Reply:
This looks great, as does the placeholder example. However, I think we could combine the two for a "Controlled Component" variant, since they're showing basically the same thing. That gives us a good place to call out that using `placeholder` and `value` require the component to be controlled, and both props can link to this variant from the props table.

Reply:
Ah, perfect - good call. OK, just made that change.